specification/columns/skudetails.md

```diff
@@ -0,0 +1,49 @@
+# SKU Details
+
+The **SKU Details** column represents a list of relevant properties shared by all resources having the same [**SKU ID**](#skuid) which are not captured in another FOCUS column. These properties enable a practitioner to more easily calculate metrics such as total units of a service when it is not directly billed in those units (e.g. cores) and thus enables FinOps capabilities such as unit economics.
+
+
+The _SkuDetails_ column adheres to the following requirements:
+
+* The properties (both key and value) contained in the _SkuDetails_ column **MUST** be shared across all resources having the same _SkuId_.
+* The _SkuDetails_ column **MUST** be in [_KeyValueFormat_](#key-valueformat).
+* The _SkuDetails_ column **SHOULD NOT** contain properties which are not applicable to the corresponding _SkuId_.
+* Where _SkuDetails_ contains a property with a value which is a numerical, the value provided **MUST** represent the value for a [_ConsumedUnit_](#consumedunit) of 1.
```

Comment:
You're right, I had grabbed the wrong column when referencing this. I blame too many open tabs.
```suggestion
* Where _SkuDetails_ contains a property with a value which is a numerical, the value provided **MUST** represent the value for a [_ConsumedQuantity_](#consumedquantity) of 1.
```